File: service-flight/src/main/java/com/goraceloty/hotelservice/hotel/entity/Transport.java
package com.goraceloty.hotelservice.hotel.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "transports")
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class Transport {
    @Id
    private Long transportID;
    private String typeOfTransport;
    private Integer numTotalSeats;
    private Integer numAvailableSeats;
    private Integer numBasePrice;
    private String cityDeparture;
    private String cityArrival;
    private LocalDateTime dateDeparture;
    private LocalDateTime dateArrival;

}

File: goraceloty-front-end/src/app/entity/Transport.ts
export class Transport {
  transportID?: number;
  typeOfTransport?: string;
  numTotalSeats?: number;
  numAvailableSeats?: number;
  numBasePrice?: number;
  cityDeparture?: string;
  cityArrival?: string;
  dateDeparture?: Date;
  dateArrival?: Date;

  constructor(init?: Partial<Transport>) {
    Object.assign(this, init);
  }
}
